src/commands/account/add_key/autogenerate_new_keypair/mod.rs

```diff
@@ -30,12 +30,18 @@ impl GenerateKeypair {
 ///Save an access key for this account
 pub enum SaveMode {
     #[strum_discriminants(strum(
-        message = "save-to-keychain   - Save automatically generated key pair to keychain"
+        message = "save-to-keychain         - Save automatically generated key pair to keychain"
     ))]
     ///Save automatically generated key pair to keychain
     SaveToKeychain(self::save_keypair_to_keychain::SaveKeypairToKeychain),
+    #[cfg(target_os = "macos")]
     #[strum_discriminants(strum(
-        message = "print-to-terminal  - Print automatically generated key pair in terminal"
+        message = "save-to-macos-keychain   - Save automatically generated key pair to OS X keychain"
+    ))]
+    ///Save automatically generated key pair to OS X keychain
+    SaveToOsxKeychain(self::save_keypair_to_keychain::SaveKeypairToKeychain),
```

Comment:
Let's move it to the top, so it is the first ("default") option on MacOS

Reply:
fixed